packages/features/data-table/components/filters/index.tsx

```diff
@@ -122,16 +123,14 @@ const ColumnVisibilityButton = forwardRef(ColumnVisibilityButtonComponent) as <T
 // Filters
 interface AddFilterButtonProps<TData> {
   table: Table<TData>;
-  externalFilters?: ExternalFilter[];
 }
```

Comment:
`externalFilters` is no longer needed as I've migrated Routing Form and People filter properly.